src/Sylius/Bundle/PromotionBundle/Resources/config/services/validators.xml

```diff
@@ -34,55 +39,66 @@
             <argument type="tagged_iterator" tag="sylius.catalog_promotion.action_validator" index-by="key" />
             <tag name="validator.constraint_validator" alias="sylius_catalog_promotion_action" />
         </service>
+        <service id="sylius.validator.catalog_promotion_action" alias="Sylius\Bundle\PromotionBundle\Validator\CatalogPromotionActionValidator" />
```

Comment:
This one is removed on 2.0:
https://github.com/Sylius/Sylius/pull/15797/files#diff-fadd2cd29fb82daa8f5462acf5bc2b54d18846ddeb5b1326d98414d93a47ef52L32